### ISSUE TITLE
Crunchyroll & Funimation Ads

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdGuardSDNSFilter#491]
@jimmysofat6864 commented on [Sun Nov 22 2020](https://github.com/AdguardTeam/AdGuardDNS/issues/122)

When I play any video in the Crunchyroll app or the Funimation app, ads go through. However, I found that by using the Energized blu ad list along with adguard dns, the ads in the crunchyroll app are gone. I have the list of domains that need to be blocked below. Can you add the energized blu ad list into Adguard DNS so crunchyroll ads are blocked?

[2020-11-21 23:53:51] 127.0.0.1 firebase-settings.crashlytics.com *.firebase-settings.crashlytics.com
[2020-11-21 23:53:51] 127.0.0.1 firebase-settings.crashlytics.com *.firebase-settings.crashlytics.com
[2020-11-21 23:53:58] 127.0.0.1 www.googleadservices.com *.www.googleadservices.com
[2020-11-21 23:53:58] 127.0.0.1 www.googleadservices.com *.www.googleadservices.com
[2020-11-21 23:54:20] 127.0.0.1 firebaselogging-pa.googleapis.com *.firebaselogging-pa.googleapis.com
[2020-11-21 23:54:20] 127.0.0.1 firebaselogging-pa.googleapis.com *.firebaselogging-pa.googleapis.com
[2020-11-21 23:54:22] 127.0.0.1 5fd74.v.fwmrm.net *.5fd74.v.fwmrm.net
[2020-11-21 23:54:22] 127.0.0.1 5fd74.v.fwmrm.net *.5fd74.v.fwmrm.net
[2020-11-21 23:54:23] 127.0.0.1 aax.amazon-adsystem.com *.aax.amazon-adsystem.com
[2020-11-21 23:54:23] 127.0.0.1 aax.amazon-adsystem.com *.aax.amazon-adsystem.com
[2020-11-21 23:55:42] 127.0.0.1 5fd74.v.fwmrm.net *.5fd74.v.fwmrm.net
[2020-11-21 23:55:42] 127.0.0.1 5fd74.v.fwmrm.net *.5fd74.v.fwmrm.net
[2020-11-21 23:55:43] 127.0.0.1 aax.amazon-adsystem.com *.aax.amazon-adsystem.com
[2020-11-21 23:55:43] 127.0.0.1 aax.amazon-adsystem.com *.aax.amazon-adsystem.com
[2020-11-21 23:57:49] 127.0.0.1 ssl.google-analytics.com *.ssl.google-analytics.com
[2020-11-21 23:57:49] 127.0.0.1 ssl.google-analytics.com *.ssl.google-analytics.com
[2020-11-21 23:57:49] 127.0.0.1 cdn.krxd.net *.cdn.krxd.net
[2020-11-21 23:57:49] 127.0.0.1 cdn.krxd.net *.cdn.krxd.net
[2020-11-21 23:57:49] 127.0.0.1 firebase-settings.crashlytics.com *.firebase-settings.crashlytics.com
[2020-11-21 23:57:49] 127.0.0.1 firebase-settings.crashlytics.com *.firebase-settings.crashlytics.com
[2020-11-21 23:57:50] 127.0.0.1 mobile-collector.newrelic.com *.mobile-collector.newrelic.com
[2020-11-21 23:57:50] 127.0.0.1 mobile-collector.newrelic.com *.mobile-collector.newrelic.com
[2020-11-21 23:57:52] 127.0.0.1 app.adjust.com *.app.adjust.com
[2020-11-21 23:57:52] 127.0.0.1 app.adjust.com *.app.adjust.com
[2020-11-21 23:58:19] 127.0.0.1 firebaselogging-pa.googleapis.com *.firebaselogging-pa.googleapis.com
[2020-11-21 23:58:19] 127.0.0.1 firebaselogging-pa.googleapis.com *.firebaselogging-pa.googleapis.com
[2020-11-21 23:58:26] 127.0.0.1 metrics.brightcove.com *.metrics.brightcove.com
[2020-11-21 23:58:26] 127.0.0.1 metrics.brightcove.com *.metrics.brightcove.com
[2020-11-21 23:58:30] 127.0.0.1 2517d.v.fwmrm.net *.2517d.v.fwmrm.net
[2020-11-21 23:58:30] 127.0.0.1 2517d.v.fwmrm.net *.2517d.v.fwmrm.net
[2020-11-21 23:58:52] 127.0.0.1 mobile-collector.newrelic.com *.mobile-collector.newrelic.com
[2020-11-21 23:58:52] 127.0.0.1 mobile-collector.newrelic.com *.mobile-collector.newrelic.com

---

@Whyabitime commented on [Sun Nov 22 2020](https://github.com/AdguardTeam/AdGuardDNS/issues/122#issuecomment-731705727)

@el-termikael 

---

@Whyabitime commented on [Sun Nov 22 2020](https://github.com/AdguardTeam/AdGuardDNS/issues/122#issuecomment-731705750)

@Reload_Admins

---

@Whyabitime commented on [Sun Nov 22 2020](https://github.com/AdguardTeam/AdGuardDNS/issues/122#issuecomment-731705763)

Duplicate of #

---

@Whyabitime commented on [Sun Nov 22 2020](https://github.com/AdguardTeam/AdGuardDNS/issues/122#issuecomment-731707782)

Yosunkayag@gmail.com 

---

@jimmysofat6864 commented on [Sun Nov 22 2020](https://github.com/AdguardTeam/AdGuardDNS/issues/122#issuecomment-731714446)

Wanted to update the block list again but I added the VRV app which is also blocked by using the [Energized Blu](https://block.energized.pro/blu/formats/domains.txt) list 

[2020-11-22 08:01:16]	127.0.0.1	firebase-settings.crashlytics.com	*.firebase-settings.crashlytics.com
[2020-11-22 08:01:16]	127.0.0.1	firebase-settings.crashlytics.com	*.firebase-settings.crashlytics.com
[2020-11-22 08:01:17]	127.0.0.1	sdk.iad-01.braze.com	*.sdk.iad-01.braze.com
[2020-11-22 08:01:17]	127.0.0.1	sdk.iad-01.braze.com	*.sdk.iad-01.braze.com
[2020-11-22 08:01:17]	127.0.0.1	app.adjust.com	*.app.adjust.com
[2020-11-22 08:01:17]	127.0.0.1	app.adjust.com	*.app.adjust.com
[2020-11-22 08:01:17]	127.0.0.1	eec.crunchyroll.com	*.eec.crunchyroll.com
[2020-11-22 08:01:17]	127.0.0.1	eec.crunchyroll.com	*.eec.crunchyroll.com
[2020-11-22 08:01:17]	127.0.0.1	firebaselogging-pa.googleapis.com	*.firebaselogging-pa.googleapis.com
[2020-11-22 08:01:17]	127.0.0.1	firebaselogging-pa.googleapis.com	*.firebaselogging-pa.googleapis.com
[2020-11-22 08:01:38]	127.0.0.1	5fd74.v.fwmrm.net	*.5fd74.v.fwmrm.net
[2020-11-22 08:01:38]	127.0.0.1	5fd74.v.fwmrm.net	*.5fd74.v.fwmrm.net
[2020-11-22 08:01:38]	127.0.0.1	aax.amazon-adsystem.com	*.aax.amazon-adsystem.com
[2020-11-22 08:01:38]	127.0.0.1	aax.amazon-adsystem.com	*.aax.amazon-adsystem.com
[2020-11-22 08:02:31]	127.0.0.1	eec.crunchyroll.com	*.eec.crunchyroll.com
[2020-11-22 08:02:31]	127.0.0.1	eec.crunchyroll.com	*.eec.crunchyroll.com
[2020-11-22 08:02:34]	127.0.0.1	eec.crunchyroll.com	*.eec.crunchyroll.com
[2020-11-22 08:02:34]	127.0.0.1	eec.crunchyroll.com	*.eec.crunchyroll.com
[2020-11-22 08:02:48]	127.0.0.1	firebaselogging-pa.googleapis.com	*.firebaselogging-pa.googleapis.com
[2020-11-22 08:02:48]	127.0.0.1	firebaselogging-pa.googleapis.com	*.firebaselogging-pa.googleapis.com
[2020-11-22 08:03:03]	127.0.0.1	crashlytics2.l.google.com	*.crashlytics2.l.google.com
[2020-11-22 08:03:03]	127.0.0.1	vrv-eec.etp-prod.com	*.etp-prod.com
[2020-11-22 08:03:03]	127.0.0.1	vrv-eec.etp-prod.com	*.etp-prod.com
[2020-11-22 08:03:05]	127.0.0.1	ssl.google-analytics.com	*.ssl.google-analytics.com
[2020-11-22 08:03:05]	127.0.0.1	ssl.google-analytics.com	*.ssl.google-analytics.com
[2020-11-22 08:03:05]	127.0.0.1	cdn.krxd.net	*.cdn.krxd.net
[2020-11-22 08:03:05]	127.0.0.1	cdn.krxd.net	*.cdn.krxd.net
[2020-11-22 08:03:05]	127.0.0.1	firebase-settings.crashlytics.com	*.firebase-settings.crashlytics.com
[2020-11-22 08:03:05]	127.0.0.1	firebase-settings.crashlytics.com	*.firebase-settings.crashlytics.com
[2020-11-22 08:03:05]	127.0.0.1	mobile-collector.newrelic.com	*.mobile-collector.newrelic.com
[2020-11-22 08:03:05]	127.0.0.1	mobile-collector.newrelic.com	*.mobile-collector.newrelic.com
[2020-11-22 08:03:05]	127.0.0.1	app.adjust.com	*.app.adjust.com
[2020-11-22 08:03:05]	127.0.0.1	app.adjust.com	*.app.adjust.com
[2020-11-22 08:03:30]	127.0.0.1	metrics.brightcove.com	*.metrics.brightcove.com
[2020-11-22 08:03:30]	127.0.0.1	metrics.brightcove.com	*.metrics.brightcove.com
[2020-11-22 08:03:33]	127.0.0.1	2517d.v.fwmrm.net	*.2517d.v.fwmrm.net
[2020-11-22 08:03:33]	127.0.0.1	2517d.v.fwmrm.net	*.2517d.v.fwmrm.net
[2020-11-22 08:03:57]	127.0.0.1	reports.crashlytics.com	*.reports.crashlytics.com
[2020-11-22 08:03:57]	127.0.0.1	reports.crashlytics.com	*.reports.crashlytics.com
[2020-11-22 08:04:04]	127.0.0.1	firebaselogging-pa.googleapis.com	*.firebaselogging-pa.googleapis.com
[2020-11-22 08:04:04]	127.0.0.1	firebaselogging-pa.googleapis.com	*.firebaselogging-pa.googleapis.com
[2020-11-22 08:04:15]	127.0.0.1	firebaselogging-pa.googleapis.com	*.firebaselogging-pa.googleapis.com
[2020-11-22 08:04:15]	127.0.0.1	firebaselogging-pa.googleapis.com	*.firebaselogging-pa.googleapis.com
[2020-11-22 08:04:17]	127.0.0.1	vrv-eec.etp-prod.com	*.etp-prod.com
[2020-11-22 08:04:17]	127.0.0.1	vrv-eec.etp-prod.com	*.etp-prod.com
[2020-11-22 08:04:36]	127.0.0.1	q4mn.adj.st	*.q4mn.adj.st
[2020-11-22 08:04:36]	127.0.0.1	q4mn.adj.st	*.q4mn.adj.st
[2020-11-22 08:04:36]	127.0.0.1	snssdk1233.onelink.me	*.snssdk1233.onelink.me
[2020-11-22 08:04:36]	127.0.0.1	snssdk1233.onelink.me	*.snssdk1233.onelink.me
[2020-11-22 08:04:36]	127.0.0.1	vrv-eec.etp-prod.com	*.etp-prod.com
[2020-11-22 08:04:36]	127.0.0.1	vrv-eec.etp-prod.com	*.etp-prod.com
[2020-11-22 08:05:51]	127.0.0.1	vrv-eec.etp-prod.com	*.etp-prod.com
[2020-11-22 08:05:51]	127.0.0.1	vrv-eec.etp-prod.com	*.etp-prod.com
[2020-11-22 08:06:03]	127.0.0.1	vrv-eec.etp-prod.com	*.etp-prod.com
[2020-11-22 08:06:03]	127.0.0.1	vrv-eec.etp-prod.com	*.etp-prod.com
[2020-11-22 08:06:56]	127.0.0.1	aax.amazon-adsystem.com	*.aax.amazon-adsystem.com
[2020-11-22 08:06:56]	127.0.0.1	aax.amazon-adsystem.com	*.aax.amazon-adsystem.com
[2020-11-22 08:06:58]	127.0.0.1	5fd74.v.fwmrm.net	*.5fd74.v.fwmrm.net
[2020-11-22 08:06:58]	127.0.0.1	5fd74.v.fwmrm.net	*.5fd74.v.fwmrm.net

**Comment by TPS** [comment]
#490?

**Comment by ameshkov** [comment]
yep, dup
